tools/stringtable_validator.py

```diff
@@ -115,6 +115,29 @@ def check_stringtable(filepath):
                 print("  ERROR: Key '{}' is defined {} times.".format(id, count))
                 errors += 1
 
+    # Check whitespace for tabs and correct number of indenting spaces
+    with open(filepath, "r", encoding = "latin-1") as file:
```

Comment:
Should this be `utf-8`?

Reply:
Yes, `utf-8` should handle all languages just fine. Unicode (Python default) will however not.

Reply:
I think `utf-8` had problems with `utf-8 bom`, or something like that. and latin just worked
but we enforce correct bom so it shouldn't be an issue